ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -439,14 +450,58 @@ gcp_file_size([[maybe_unused]] WT_FILE_HANDLE *file_handle, WT_SESSION *session,
 static int
 gcp_object_size(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, wt_off_t *sizep)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(name);
-    WT_UNUSED(sizep);
+    bool exists;
+    size_t size;
+    gcp_file_system *fs = reinterpret_cast<gcp_file_system *>(file_system);
+    int ret = fs->gcp_conn->object_exists(name, exists, size);
+    if (ret != 0) {
+        std::cerr << "gcp_object_size: GetObjectMetadata request to google cloud failed."
+                  << std::endl;
+        return (ret);
```

Comment:
no brackets around return values in cpp

Reply:
Added [4cc6a5e](https://github.com/wiredtiger/wiredtiger/pull/8794/commits/4cc6a5efcbf04cc58160032f6e533aec9aa3a406)